### ISSUE TITLE
Suggestion about GenDustry

### ISSUE THREAD

**Issue by IyaTaisho** [FTBTeam/FTB-University#319]
I noted while playing through FTB University early on (started late last year) that Forestry had quests for bees.  However, I noted they were optional and halfway did them (up to the point you need bees actually).  GenDustry is of course gated behind the bee quests, which makes sense.

Cut to now where I am in the End Game quest line and to my surprise, Liquid DNA is needed...

If something is decided to be optional, make it actually optional.  My suggestion, get rid of the Bucket of Liquid DNA in the quest.  Otherwise, require the bee quests.  I could have done bees and had them up to proper levels to complete this quest if not for the fact it was said the bees were optional.

**Comment by aaronhowser1** [comment]
Fair enough, the decision to make them optional was _way_ before I even thought of making recipes for creative items.
